### ISSUE TITLE
Hide "View Rotation Locked" message in Godot 4

### ISSUE THREAD

**Issue by elvisish** [godotengine/godot-proposals#7972]
### Describe the project you are working on

Walking simulator.

### Describe the problem or limitation you are having in your project

I keep view rotation locked so I can use middle-mouse button to pan instead of rotate in ortho mode, however Godot 4 added this message to the bottom of the screen:
![image](https://github.com/godotengine/godot-proposals/assets/16231628/36330efc-92c2-4976-84e5-d85f410ee3f8)

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

An option to hide the message for those of us that always have Lock View Rotation enabled.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

An option in editor settings.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

Not that I can imagine.

### Is there a reason why this should be core and not an add-on in the asset library?

It's a basic courtesy option.

**Comment by Calinou** [comment]
I wouldn't remove the message entirely, as it's important to avoid support requests with people forgetting they have the setting enabled :slightly_smiling_face: 

However, I'd change it so the message is only visible when panning.

**Comment by elvisish** [comment]
> I wouldn't remove the message entirely, as it's important to avoid support requests with people forgetting they have the setting enabled 🙂
> 
> However, I'd change it so the message is only visible when panning.

Well, I don't think an editor setting would make much difference as support request people wouldn't probably toggle it if they didn't know what it did? Personally, I'm always jumping between panning and switching perspectives so it would be equally annoying to have it appear when panning (more so probably) and toggling it as an advanced setting would be a nice complementary legacy setting for those of us used to Godot 3.x's lack of annoying message.

**Comment by thygrrr** [comment]
I simply rebound my pan action in the settings to use middle mouse.

**Comment by elvisish** [comment]
> I simply rebound my pan action in the settings to use middle mouse.

No, because I only want pan mode in ortho, and locking view rotation allows me to have both, it works perfectly well as it did in 3.x just with this annoying message at the bottom that really should be optionally displayed (as I say, an advanced editor setting that only would be changed if you know what you're doing).
